### DISCUSSION TITLE
Informations organisation questionnaires

### DISCUSSION THREAD

**Discussion by elliotvaucher** [edunumsec2/book#10] [Organisation]
Prière de répondre en copiant ce template et en remplissant les champs suivants. 

* nom enseignant.e : 
* nom gymnase : 
* canton : 
* liste de vos classes (nom - jour -heure - nb élève - orientation classe si pertinente (ie. Math/physique, économie, mixte)) :

**Comment by G-Racine** [comment]
nom enseignant.e : Racine Gilles
nom gymnase : Bugnon
canton : Vaud
liste de vos classes **pour le projet pilote**: 

- 1M4, Mardi, 13H30, 12 élèves, Bio-chimie
- 1M6, Jeudi, 13H30, 12 élèves, Mixte

**Reply by farenc-c** [reply]
top ! merci Gilles

**Comment by info-edu** [comment]
- nom enseignant.e : Amaury Dame
- nom gymnase : Provence
- canton : Vaud
- liste de vos classes: 1M1, Mardi, 13h30, 12 élèves, Bio-chimie

**Comment by cvangenot** [comment]
nom enseignant.e : Christelle Vangenot
nom gymnase : Burier
canton : Vaud
liste de vos classes : 
1M2 : vendredi, 10h15, 22 élèves (en demi-groupe tous les 15 jours), Biologie-Chimie
1M5 : mardi, 13h35, 21 élèves (en demi-groupe tous les 15 jours), Biologie-Chimie

**Comment by emiliemichaud** [comment]
nom enseignante : Emilie Michaud
nom gymnase : Gymnase de Nyon La Cote (GNLC)
canton : Vaud 
liste de vos classes (nom - jour -heure - nb élève - orientation classe si pertinente (ie. Math/physique, économie, mixte)) : 1M18 , 1M19, 1M20, 1M21 en demi groupe tous les 15 jours (j'éditerai ultérieurement les options)

**Comment by Rebinette** [comment]
nom enseignante : Julia Rebstein
nom gymnase : Gymnase du Bugnon (site de Sévelin)
canton : Vaud
liste de vos classes (nom - jour -heure - nb élève - orientation classe si pertinente (ie. Math/physique, économie, mixte)) : 1M02 - jeudi  14h10-15h50 - 21 élèves (en demi groupe tous les 15 jours) - OS biol-chimie

**Comment by Andrea-Franco-08** [comment]
nom enseignant : Andrea Franco
nom gymnase : Gymnase de Renens
canton : Vaud
liste de vos classes  : 
1M05 - mardi - 13h45-15h25 - 24 élèves - OS physique et applications des mathématiques
1M09 - mardi - 8h15-9h55 - 26 élèves - OS mixte

**Comment by rasql** [comment]
nom enseignant.e : Raphael Holzer
nom gymnase : Bugnon
canton : Vaud
liste de vos classes (nom - jour -heure - nb élève) :
1M1 - lundi - 9h05 - 25 élèves
1M5 - lundi - 12h45 - 25 élèves
1M4 - lundi - 14h35 - 23 élèves
1M6 - mercreid - 10h10 -25 élèves

**Comment by Dalker** [comment]
- **nom enseignant.e** : Daniel Kessler
    - **nom gymnase** : CEC André-Chavanne
    - **canton** : Genève
    - **liste des classes où du matériel EdunumSec2 sera employé**:
      - 1IN.DF.03 - lundi 15:15-16:45  - 16 élèves
      - 1IN.DF.04 - jeudi 15:15-16:45  - 16 élèves

**Comment by juancarlosbarros** [comment]
nom enseignant.e : Juan-Carlos Barros
nom gymnase : Collège de Candolle
canton : Genève
liste de vos classes : 

- 1INDF01 - mercredi - H1H2 - 16 élèves - pas d'orientation
- 1INDF02 - mercredi - H3H4 - 16 élèves - pas d'orientation
- 1INDF03 - jeudi - H3H4 - 16 élèves - pas d'orientation

JC

**Comment by espurs** [comment]
nom enseignant.e : Ivan Moura
    nom gymnase : Gymnase de Burier
    canton : Vaud
    liste de vos classes : 
    - 1M6 - vendredi 14:30 - 16:05 - 22 élèves  (demi classe en alternance) - OS Biologie-chimie
    - 1M7 - mercredi 10:00-11:50 - 21 élèves (demi classe en alternance) - OS Physique-Math

**Comment by lio-ch** [comment]
Mieux vaut tard que jamais, désolé :)

nom enseignant.e : Lionel Chatelain
nom gymnase : Gymnase de Chamblandes
canton : VD
classes pilotes :
- 1m4 (Math Physique) - 22 élèves - Mardi par demi groupe, 2 périodes, 8h15-9h55
- 1m6  - 24 élèves - mercredi par demi groupe, 2 périodes, 10h15-11h55
- 1m2 (bio chimie) - 22 élèves - jeudi par demi groupe, 2 périodes, 8h15-9h55
- 1m8  - 24 élèves - jeudi par demi groupe, 2 périodes, 10h15-11h55

**Comment by carlabastos** [comment]
nom enseignant.e : Carla Bastos
    nom gymnase : GNLC
    canton : VD
    liste de vos classes (nom - jour -heure - nb élève - orientation classe si pertinente (ie. Math/physique, économie, mixte)) :

-     1M22 - 23 élèves - Mardi par demi groupe, 2 périodes, 10h05-11h40
-     1M23 - 22 élèves - Lundi par demi groupe, 2 périodes, 15h00-16h35
